package.json

```diff
@@ -7,8 +7,10 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "eslint ./src --ext .js,.ts",
+    "lint-fix": "eslint ./src --fix --ext .js,.ts",
     "gen": "ng-openapi-gen --input http://localhost:8080/atlas/v3/api-docs -o generated/api",
     "postgen": "prettier generated/api --write",
+    "gen-api": "ng-openapi-gen --input http://localhost:8080/atlas/v3/api-docs -o generated/api && prettier generated/api --write",
```

Comment:
Why did you add this?
gen & postgen should already handle this.

Reply:
Laziness and to ensure both commands have been run.